client/transaction-pool/src/tests/mod.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018-2020 Parity Technologies (UK) Ltd.
+// Copyright 2020 Parity Technologies (UK) Ltd.
```

Comment:
I think should stay 2018-2020, but I don't care enough :)

```suggestion
// Copyright 2018-2020 Parity Technologies (UK) Ltd.
```

Reply:
It's just GitHub thinks the file moved, while it is in fact a new file completely